src/emulator/util/include/util/string_convert.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <vector>
+#include <codecvt>   // std::codecvt_utf8
+
+using ProgramArgsWide = std::vector<std::wstring>;
+
+// convert UTF-8 string to wstring
+inline std::wstring utf_to_wide(const std::string& str)
```

Comment:
I'd probably move these into the .cpp file. They're not templates and I wouldn't think they're called enough to warrant inlining.

That'll mean we can move the `<codecvt>` include to the .cpp file too.

I think the macOS build may have failed because `<locale>` wasn't included.

Reply:
>I'd probably move these into the .cpp file.
To be honest I did, but I was getting unresolved symbol errors that I wasn't sure how how to fix so I moved them back haha. I'll try again and ask you on Discord if I can't figure it out.

>I think the macOS build may have failed because <locale> wasn't included.
Yeah I fixed that and another bug on the unix code path.